metrics.nim

```diff
@@ -108,6 +108,12 @@ when defined(metrics):
       for metric in metrics:
         result.add(metric.toText())
 
+template value*(collector: Collector, labelValues: Labels = @[]): float64 =
+  when defined(metrics):
+    collector.metrics[labelValues][0].value
+  else:
+    0.0
```

Comment:
Are there any useful metrics for which 0 isn't a sensible default value?

Reply:
This is not a default value. It's a "metrics are disabled" value.